runtime/compiler/build/files/common.mk

```diff
@@ -380,6 +379,7 @@ JIT_PRODUCT_SOURCE_FILES+=\
     compiler/runtime/ValueProfiler.cpp \
     compiler/runtime/codertinit.cpp \
     compiler/runtime/emfloat.c \
+    omr/compiler/control/CompilationController.cpp \
```

Comment:
Sorry, just discovered this.  You will also need to add the new file `compiler/control/J9CompilationStrategy.cpp` to this file.

As this does not affect builds in the open project I don't need to re-run all the tests and will just re-run a private build instead.

Reply:
Change updated, pls take another look.